tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -83,18 +84,25 @@ def setup(self):
             }
 
     @pytest.fixture()
-    def base_setup(self, interface_iterate, storageclass_factory, pod_factory):
+    def base_setup(self, interface_type, storageclass_factory, pod_factory):
         """
         A setup phase for the test
 
         Args:
-            interface_iterate: A fixture to iterate over ceph interfaces
+            interface_type: A fixture to iterate over ceph interfaces
             storageclass_factory: A fixture to create everything needed for a
                 storageclass
             pod_factory: A fixture to create new pod
         """
-        self.interface = interface_iterate
-        self.sc_obj = storageclass_factory(self.interface)
+        self.interface = interface_type
+        if self.interface == constants.CEPHBLOCKPOOL_THICK:
+            self.sc_obj = storageclass_factory(
+                interface=constants.CEPHBLOCKPOOL,
+                new_rbd_pool=True,
+                rbd_thick_provision=True,
+            )
+        else:
```

Comment:
`self.interface = interface_iterate` should be included in the else block to run tests with cephfs & rbd interface

Reply:
@keesturam - regarding the label - no problem. I'll remove the new labels from this PR and leave only "performance" label for all the test cases. And in the next PR we can add new labels to the test cases that we decide to run with other than "performance" label.

Reply:
> `self.interface = interface_iterate` should be included in the else block to run tests with cephfs & rbd interface

@keesturam Currently self.interface=interface_type line appears before the if--else-- and will be executed for all 3 interfaces. Thanks